hazelcast-spring/src/main/resources/hazelcast-spring-4.1.xsd

```diff
@@ -1574,6 +1574,7 @@
                     <xs:element name="reliable-topic" type="client-reliable-topic" minOccurs="0" maxOccurs="unbounded"/>
                     <xs:element name="metrics" type="client-metrics" minOccurs="0" maxOccurs="unbounded"/>
                     <xs:element name="instance-tracking" type="instance-tracking" minOccurs="0" maxOccurs="1"/>
+                    <xs:element name="native-memory" type="native-memory" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Should be: `maxOccurs="1"`

Reply:
Can we make it like that then, at least for the newly added config? Or are you making it unbounded to be in line with regular XML config?

Reply:
Done it. It's only informal though since the number of occurrences for the top-level elements is not bounded, so one can still write any number of `metrics` elements, such as `cluster-data`, ...